collections/ansible_collections/demo/cloud/roles/aws/templates/aws_windows_userdata.j2

```diff
@@ -17,7 +17,7 @@ reg delete "HKLM\SOFTWARE\Policies\Microsoft\Windows\WinRM\Service" /v DisableRu
 Set-MpPreference -DisableRealtimeMonitoring $true
 
 # Enable WinRM
-Invoke-WebRequest -Uri https://raw.githubusercontent.com/ansible/ansible/devel/examples/scripts/ConfigureRemotingForAnsible.ps1 -OutFile C:\ConfigureRemotingForAnsible.ps1
+Invoke-WebRequest -Uri https://raw.githubusercontent.com/ansible/ansible-documentation/devel/examples/scripts/ConfigureRemotingForAnsible.ps1 -OutFile C:\ConfigureRemotingForAnsible.ps1
```

Comment:
same as above, i don't think this role is being called so the changes to userdata here won't be used.  unfortunately there doesn't seem to be an easy way to override the userdata template location in the [aws.infrastructure_config_demos.create_vm](https://github.com/ansible-content-lab/aws.infrastructure_config_demos/tree/main/roles/create_vm) role without changing it upstream in that collection.

Reply:
see #78, the ConfigureRemotingForAnsible.ps1 script being pulled in by the aws.infrastructure_config_demos.create_vm role is essentially the same as your updated one here so i don't think this change is needed.

Reply:
@jce-redhat good call on these two, I think I went down this rabbit hole before I realized we were using the external cloud content repo. In fact, I think I do this everytime 😅 I will revert these two changes.

Reply:
removing